### ISSUE TITLE
vcpkg cimg set_property can not be used on an ALIAS target.

### ISSUE THREAD

**Issue by ResearchDaniel** [inviwo/inviwo#979]
CMake Error at vcpkg_installed/x64-windows/share/tiff/vcpkg-cmake-wrapper.cmake:7 (set_property):
  set_property can not be used on an ALIAS target.
Call Stack (most recent call first):
  C:/Users/danjo/Documents/src/vcpkg/vcpkg/scripts/buildsystems/vcpkg.cmake:360 (include)
  modules/cimg/CMakeLists.txt:39 (find_package)

**Comment by ResearchDaniel** [comment]
Ah, right. Need to disable those when using the vpkg toolchain. Could we detect it perhaps?

**Comment by petersteneteg** [comment]
can you elaborate? what did we try and set?

**Comment by ResearchDaniel** [comment]
If I use the vcpkg config and do not set the USE_EXTERNAL setting for cimg I get this error. I have not tried without the vcpkg so perhaps the error is the same without. It should have to do with the internal cimg cmake file.

Also, our assimp configuration seem to be a bit different from the vcpkg one in terms on target naming (vcpkg append the compiler version to the library but we do not)

**Comment by petersteneteg** [comment]
still not sure I understand what you mean. what to do mean with vcpkg config? setting the toolchain file? have you also "integrated" vcpkg? do you have cimg installed in vcpkg? the errors seems to come from finding TIFF? Why does the assimp target names matter? you always link assimp::assimp?

**Comment by ResearchDaniel** [comment]
Yes, the tool chain file. I have installed cimg in vcpkg (it works when I use the external cimg). The error only occurs when configuring cmake with USE_EXTERNAL_CIMG = off 

Assimp: I’m not sure why it matters, perhaps some cmake cache thingy or related to the issue I linked to in the other issue. The suggested solution of upgrading visual studio did not work so I had to use our internal assimp
